src/outputs/history.cpp

```diff
@@ -24,44 +24,59 @@
 #include "../athena.hpp"
 #include "../athena_arrays.hpp"
 #include "../coordinates/coordinates.hpp"
+#include "../cr/cr.hpp"
 #include "../field/field.hpp"
 #include "../globals.hpp"
 #include "../gravity/gravity.hpp"
 #include "../hydro/hydro.hpp"
 #include "../mesh/mesh.hpp"
-#include "../orbital_advection/orbital_advection.hpp"
+#include "../nr_radiation/radiation.hpp"
 #include "../scalars/scalars.hpp"
 #include "outputs.hpp"
 
 // NEW_OUTPUT_TYPES:
 
 // "3" for 1-KE, 2-KE, 3-KE additional columns (come before tot-E)
-#define NHISTORY_VARS ((NHYDRO) + (SELF_GRAVITY_ENABLED > 0) + (NFIELD) + 3 + (NSCALARS))
+//14 radiation variables,
+// if no RADIATION_ENABLED, they are always 0
+#if ((NR_RADIATION_ENABLED > 0) || (IM_RADIATION_ENABLED > 0))
+  #define NRAD (14)
+#else
+  #define NRAD 0
+#endif
+
+#if (CR_ENABLED > 0)
+  #define NCRV (4)
+#else
+  #define NCRV 0
+#endif
+
+#define NHISTORY_VARS ((NHYDRO) + (SELF_GRAVITY_ENABLED) + (NFIELD) + 3 + (NSCALARS))
```

Comment:
this line seems incorrect. Should keep `(SELF_GRAVITY_ENABLED > 0) ` term? And shouldnt `NCRV`, `NRAD` appear?

Reply:
I see you add `NCRV`, `NRAD` in the loop bounds outside of `NHISTORY_VARS`. Any reason for that design choice?